uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -39,6 +39,41 @@ Feature: Greengrass V2 LogManager
         """
         And I deploy the Greengrass deployment configuration
         Then the Greengrass deployment is COMPLETED on the device after 2 minutes
-        And I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
-        And I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
-        And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
\ No newline at end of file
+        # And I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
+        Then I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
+        And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
+
```

Comment:
I think here you can set the `deleteLogFileAfterCloudUpload` to false and check in a step that the files are not deleted.

Reply:
Why is this resolved?

Reply:
I have tested setting it as false as well

Reply:
That should be a test here.

Reply:
Oh ok got it